By Nishant Limbachia <nishant@mnspace.net>
1. Fixup paths for SlackBuilds.org
2. Tighten default access mode

--- tools/httpd_conf.orig	2009-01-27 10:41:54.000000000 -0600
+++ tools/httpd_conf	2009-01-27 15:04:31.000000000 -0600
@@ -1,23 +1,19 @@
 #
-# Content of this file, with correct values, can be automatically added to
-# your Apache server by using the AWStats configure.pl tool.
-#
-
-#
 # Directives to add to your Apache conf file to allow use of AWStats as a CGI.
-# Note that path "/usr/local/awstats/" must reflect your AWStats install path.
 #
-Alias /awstatsclasses "/usr/local/awstats/wwwroot/classes/"
-Alias /awstatscss "/usr/local/awstats/wwwroot/css/"
-Alias /awstatsicons "/usr/local/awstats/wwwroot/icon/"
-ScriptAlias /awstats/ "/usr/local/awstats/wwwroot/cgi-bin/"
+
+Alias /awstatsclasses "@DOCROOT@/awstats/wwwroot/classes/"
+Alias /awstatscss "@DOCROOT@/awstats/wwwroot/css/"
+Alias /awstatsicons "@DOCROOT@/awstats/wwwroot/icon/"
+ScriptAlias /awstats/ "@DOCROOT@/awstats/wwwroot/cgi-bin/"
 
 #
 # This is to permit URL access to scripts/files in AWStats directory.
 #
-<Directory "/usr/local/awstats/wwwroot">
+<Directory "@DOCROOT@/awstats/wwwroot">
     Options None
     AllowOverride None
-    Order allow,deny
-    Allow from all
+    Order deny,allow
+    Deny from all
+    Allow from localhost
 </Directory>
